libs/langchain/langchain/embeddings/johnsnowlabs.py

```diff
@@ -0,0 +1,96 @@
+import os
+import sys
+from typing import Any, List
+
+from langchain.embeddings.base import Embeddings
+from langchain.pydantic_v1 import BaseModel, Extra
+
+
+class JohnSnowLabsEmbeddings(BaseModel, Embeddings):
+    """JohnSnowLabs embedding models
+
+    To use, you should have the ``johnsnowlabs`` python package installed.
+    Example:        .. code-block:: python
+            from langchain.embeddings.johnsnowlabs import JohnSnowLabsEmbeddings
+            document = "foo bar"
+            embedding = JohnSnowLabsEmbeddings('embed_sentence.bert')
+            output = embedding.embed_query(document)
+    """
+
+    async def aembed_documents(self, texts: List[str]) -> List[List[float]]:
```

Comment:
this and aembed_query have default implementations, any reason not to just keep those?